src/test/java/uk/gov/companieshouse/web/companydocuments/service/CompanyServiceTest.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.companieshouse.web.companydocuments.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.company.CompanyResourceHandler;
+import uk.gov.companieshouse.api.handler.company.request.CompanyGet;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
+import uk.gov.companieshouse.web.companydocuments.api.ApiClientService;
+import uk.gov.companieshouse.web.companydocuments.exception.ServiceException;
+import uk.gov.companieshouse.web.companydocuments.model.CompanyDetail;
+import uk.gov.companieshouse.web.companydocuments.transformer.CompanyTransformer;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CompanyServiceTest {
+
+    @Mock
+    private ApiClient apiClient;
+
+    @Mock
+    private ApiClientService apiClientService;
+
+    @Mock
+    private CompanyResourceHandler companyResourceHandler;
+
+    @Mock
+    private CompanyGet companyGet;
+
+    @Mock
+    private ApiResponse<CompanyProfileApi> responseWithData;
+
+    @Mock
+    private CompanyProfileApi companyProfile;
+
+    @Mock
+    private CompanyDetail companyDetail;
+
+    @Mock
+    private CompanyTransformer companyTransformer;
+
+    @InjectMocks
+    private CompanyService companyService;
+
+    private static final String COMPANY_NUMBER = "companyNumber";
```

Comment:
Constants should come first.

Reply:
Moved in [b916061](https://github.com/companieshouse/company-documents.web.ch.gov.uk/pull/10/commits/b9160619da9c40be5d6b3314aec0733f77b248a6) 👍